packages/server-admin-ui/src/views/DataBrowser/DataBrowser.js

```diff
@@ -396,23 +396,27 @@ class DataBrowser extends Component {
                       </tr>
                     </thead>
                     <tbody>
-                      {Object.keys(this.state.meta[this.state.context] || {})
+                      {Object.keys(this.state.data[this.state.context] || {})
                         .filter((key) => {
                           return (
                             !this.state.search ||
                             this.state.search.length === 0 ||
-                            key.indexOf(this.state.search) !== -1
+                            key
+                              .toLowerCase()
+                              .indexOf(this.state.search.toLowerCase()) !== -1
                           )
                         })
                         .sort()
                         .map((key) => {
-                          const meta = this.state.meta[this.state.context][key]
-                          const path = key
+                          const data = this.state.data[this.state.context][key]
+                          const meta =
+                            this.state.meta[this.state.context][data.path]
+                          const path = data.path
```

Comment:
Why do we need to get data by key that is path and then get the path from the data, instead of `path = key`? Or to make it simpler rename the map handler parameter `path`?